content/v2.0/monitor-alert/templates/infrastructure/docker.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Monitor Docker
+description: >
+  Use the [Docker Monitoring template](https://https://github.com/influxdata/community-templates/tree/master/docker) to monitor your Docker containers.
+menu:
+  v2_0:
+    parent: Infrastructure
+    name: Docker
+weight: 202
+---
+
+Use the [Docker Monitoring template](https://https://github.com/influxdata/community-templates/tree/master/docker) to monitor your Docker containers. First, [apply the template](apply-the-template), and then [view incoming data](view-incoming-data).
+This template uses the [Docker input plugin](/v2.0/reference/telegraf-plugins/#docker) to collect metrics stored in InfluxDB and display these metrics in a dashboard.
+
+The Docker Monitoring template includes the following:
+
+- one dashboard: **Docker**
+- one bucket: `docker, 7d retention`
+- labels: Telegraf plugin labels
+- one Telegraf configuration: Docker input plugin
+- one variable: `bucket`
+- four alerts: `Container cpu`, `mem`, `disk`, `non-zero exit`
```

Comment:
Notice we don't define the term "alert" --we refer to creating a "check" to alert. Thoughts?